src/components/CommonGridBasedForm/autoComplete/index.js

```diff
@@ -1,9 +1,16 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
 
 import { Autocomplete } from '@mui/material';
 import { TextField } from '@mui/material';
 const AutoComplete = (props) => {
-  const { values, onChange, index, value, options, label, placeholder } = props;
+  const { values, onChange, index, value, options, placeholder } = props;
+
+  const labelValue = value
+    ? values.filter(({ id }) => {
+        return id === value;
+      })
+    : '';
```

Comment:
you can  use And condition here

Reply:
May be he wants to have empty string here but let him confirm

Reply:
initially i am getting value undefined so i am saying that if and only if i am getting truthy value then set the label else i am giving empty string so value will not become undefined at autocomplete tag.